index.bs

```diff
@@ -11,6 +11,7 @@ Editor: Pete Snyder, w3cid 109401, Brave https://brave.com, psnyder@brave.com
 Former Editor: Jason Novak, Apple Inc., https://apple.com
 Former Editor: Lukasz Olejnik, Independent researcher, https://lukaszolejnik.com
 Former Editor: Mike West, Google Inc., mkwst@google.com
+Former Editor: Yan Zhu, Yahoo Inc., yzhu@yahoo-inc.com
```

Comment:
I can ask Yan what email she prefers, but yan@brave.com might be better (or at least more current)

Reply:
i'm yan@brave.com, thanks!

Reply:
Yes, you're at Brave now, but when you were an editor you were at Yahoo, yeah? I think this should reflect the state at the time. Like, if @mikewest ever leaves Google, we wouldn't retroactively change his affiliation here.

Reply:
Fixed in b80f7f2.